src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -932,3 +978,87 @@ button.link {
     max-width: 200px;
     width: 100%;
 }
+
+/* Right Panel */
+
+.RoomInfo {
+    background: rgba(245, 245, 245, 0.90);
+    padding: 10px;
+}
+
+.RoomAvatar {
+    margin-top: 44px;
+}
+
+.RoomId {
+    color: #737D8C;
+    font-size: 12px;
+}
+
+.SidebarRow_collection{
+    margin-top: 36px;
+    width: 100%;
+}
+
+.RoomName h2 {
+    margin-bottom: 4px;
+    font-size: 1.8rem;
+}
+
+.SidebarRow {
+    width: 90%;
```

Comment:
Wonder what this is for? It is adding space on the right, making it look somewhat unbalanced:
![image](https://user-images.githubusercontent.com/274386/120623146-fdd4b180-c44e-11eb-92c8-945f390d0736.png)